src/firebase/init.js

```diff
@@ -8,40 +8,49 @@ import "firebase/analytics";
 
 /* Production Config */
 
-const config = {
-  apiKey: "AIzaSyB46JWorhyNJDzG20J0dujbIY46zNudibc",
-  authDomain: "auth.buspark.app",
-  projectId: "spark-project-engage",
-  storageBucket: "spark-project-engage.appspot.com",
-  messagingSenderId: "95521899365",
-  appId: "1:95521899365:web:faf942f1ef49f28af8178c",
-  measurementId: "G-L9TYR85MP5"
-};
+// const config = {
+//   apiKey: "AIzaSyB46JWorhyNJDzG20J0dujbIY46zNudibc",
+//   authDomain: "auth.buspark.app",
+//   projectId: "spark-project-engage",
+//   storageBucket: "spark-project-engage.appspot.com",
+//   messagingSenderId: "95521899365",
+//   appId: "1:95521899365:web:faf942f1ef49f28af8178c",
+//   measurementId: "G-L9TYR85MP5"
+// };
 
 /* Staging Config */
 
+const config = {
+  apiKey: "AIzaSyCRAkouqVNtyTWclJmg90BxEDIynahqOtM",
+  authDomain: "spark-project-engage-staging.firebaseapp.com",
+  projectId: "spark-project-engage-staging",
+  storageBucket: "spark-project-engage-staging.appspot.com",
+  messagingSenderId: "617899707305",
+  appId: "1:617899707305:web:c98bef1def8c3734ed92b8"
+};
+
```

Comment:
Can you change this back to the prod config?

Reply:
Will do, thanks!